src/compiler/transformers/transform-utils.ts

```diff
@@ -156,7 +156,10 @@ const objectToObjectLiteral = (obj: { [key: string]: any }, refs: WeakSet<any>):
  * @param returnExpression a TypeScript AST node to return from the getter
  * @returns an AST node representing a getter
  */
-export const createStaticGetter = (propName: string, returnExpression: ts.Expression): ts.GetAccessorDeclaration => {
+export const createStaticGetter = (
+  propName: StencilStaticGetter,
```

Comment:
anyone know if there are other functions where we should change `string` to `StencilStaticGetter`?

Reply:
I'm sure there's more, but `getStaticGetter` could use it. It's only used in testing, but types anywhere are good. Only issue is it gets called [here](https://github.com/ionic-team/stencil/blob/main/src/compiler/transformers/test/parse-encapsulation.spec.ts#L15) with a "mode" property which doesn't look like it's in the array of valid names.

I couldn't find a call to `createStaticGetter` with a "mode" property name though, so that test file could just be wrong.

Reply:
Yeah, I saw that usage and I wasn't sure if that function really does what it says it does...